src/stylesheets/base/_form.scss

```
@@ -0,0 +1,73 @@
+.form {
+  width: 100%;
+
+  &__body {
+    display: grid;
+    margin-top: 2.5rem;
+    row-gap: 2rem;
+    column-gap: 1.5rem;
+
+    @media only screen and (min-width: map-get($grid, 'small')) {
+      grid-template-columns: repeat(6, minmax(0, 1fr));
+    }
+  }
+
+  &__label {
+    display: block;
+    font-size: 0.875rem;
+    font-weight: 500;
+    color: #111827;
+    line-height: 1.5rem;
+  }
+
+  &__input {
+    display: block;
+    padding: {
+      left: 0.25rem;
+      top: 0.375rem;
+      bottom: 0.375rem;
+    }
+    border: {
+      radius: 0.375rem;
+      width: 0px;
+    }
+    width: 100%;
+    color: #111827;
+    @layer components {
```

Is the `@layer` needed? This new CSS spec aims to allow more control on the cascading order but since the component is defined with BEM, we might not need it:

```suggestion
    @apply shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600;
```

---

To use the tailwind apply utility  it is added. If directly the apply utils is added it is generating error. On the documentation of tailwind to it is recommended to use the apply util with wrap of layer components.

---

Understood 👍